src/renderer/lines.rs

```diff
@@ -105,7 +105,8 @@ impl<'a> Lines<'a> {
     }
 }
 
-/// Create a rectangle between two cells.
+/// Create a rectangle that starts on the left of `start` and ends on the right
+/// of `end` based on the given flags and size metrics.
```

Comment:
That sounds pretty clear to me! Just some minor nitpicks, then let's ship it.

`flag` should be singular since it's never all cell flags that are passed, but just the underline OR the strikeout flag. So even if a cell has both strikeout and underline, these are handled separately since they require to distinct rectangles.

```suggestion
 /// of `end`, based on the given flag and size metrics.
```